.github/workflows/create_tag_release.yml

```
@@ -33,7 +33,8 @@ jobs:
       library_name: ${{ github.event.repository.name }}
       library_version: ${{ needs.cmakelists-version.outputs.LIBRARY_VERSION }}
       github_sha: master
-      profile: release
+      profile_host: release
+      profile_build: release
```

No need to change anything here. (Even if there were that parameter,) there is no need to force the `profile:build`, it is provided by the worker so it matches its own architecture.

```suggestion
      profile: release
```